signing/sawtooth_signing/secp256k1_signer.py

```diff
@@ -184,7 +111,7 @@ def _decode_pubkey(serialized_pubkey, encoding_format='hex'):
 
 
 def generate_identifier(pubkey):
```

Comment:
Okay.  Since that whole dir is going away....don't know if we need that any more.  I thought that we were using sign public signer key as the "identifier" now....